tests/e2e/specs/web-terminal/Readme.md

```diff
@@ -0,0 +1,53 @@
+# Test Scenario for Web Terminal in OpenShift
+
+### Prerequisites:
+* Test Environment: Node.js + Webdriver + Mocha framework.
+* Automation Requirement: A browser for interacting with the OpenShift Web Console.
+* Permissions: Admin access to the test cluster.
+* Setup: Ensure the DevWorkspace Operator is preinstalled before executing the tests.
+## Scenario for WebTerminal under Admin user
+#### Note! This scenario also is used for interop testing
+1. Login as Admin:
+*  Login as admin user into OpenShift Web Console and refresh the page
+2. Clean Up Existing Resources:
+* Remove any existing DevWorkspace custom resources (`oc delete dw --all -n dedicated_namespace`)
+3. Launch WebTerminal:
+*  Click on the "WebTerminal" icon
+* Wait for the WebTerminal widget to appear
+4. Create a WebTerminal Session:
+* Click the "Create" button.
+* Wait for the WebTerminal xterm text area to appear.
+
+5. Verify Admin Session:
+* Execute the command oc whoami and verify that the session is running under the admin user.
+* Since the terminal is canvas-based and difficult to read with WebDriver, read the DevWorkspace object, parse the YAML, and extract the username.
```

Comment:
did you mean "and extract the username manually"?

Reply:
It means we get webterminal object in YAML format (oc get dw -n <some-namespace> -o yaml), parse it, and make sure, that we have been logged in as the proper user.